src/test/java/com/datatorrent/netlet/DefaultEventLoopTest.java

```diff
@@ -300,4 +300,88 @@ public void registered(SelectionKey key)
     assertFalse(client2.isConnected());
   }
 
+  @Test
+  public void startStop() throws Exception
+  {
+    assertTrue(defaultEventLoop.isActive());
+    assertEquals(thread, defaultEventLoop.start());
+    defaultEventLoop.stop();
+    assertTrue(defaultEventLoop.isActive());
+    defaultEventLoop.stop();
+    thread.join();
+    assertFalse(defaultEventLoop.isActive());
+  }
+
+  @Test
+  public void startWithExtraStop() throws Exception
+  {
+    assertTrue(defaultEventLoop.isActive());
+    defaultEventLoop.stop();
+    thread.join();
```

Comment:
It can be achieved by using Timeout Rule, there is no need to timeout individual Test methods.